manticore/core/executor.py

```diff
@@ -507,8 +509,8 @@ def generate_testcase(self, state, message = 'Testcase generated'):
                 stdout += ''.join(map(str, data))
             if sysname in ('_transmit', '_write') and fd == 2:
                 stderr += ''.join(map(str, data))
-        file(self._getFilename('test_%08x.stdout'%test_number),'a').write(stdout)
-        file(self._getFilename('test_%08x.stderr'%test_number),'a').write(stderr)
+        open(self._getFilename('test_{:08x}.stdout'.format(test_number)),'a').write(stdout)
```

Comment:
on second thought, this is really low priority, and perhaps not worth anyone's time right now